zulipterminal/themes/_template.py

```diff
@@ -104,11 +104,14 @@ class DefaultColor(Enum):
     'popup_category'  : 'bold',
     'unread_count'    : 'bold',
     'starred_count'   : '',
+    'table_head'      : 'bold',
     'filter_results'  : 'bold',
     'edit_topic'      : 'standout',
     'edit_tag'        : 'standout',
     'edit_author'     : 'bold',
     'edit_time'       : 'bold',
+    'current_user'    : '',
+    'muted'           : 'bold',
```

Comment:
It would be better to improve the test for this and add these earlier rather than later. That test change would be a bugfix of its own.

Then the migration of the test to the new form would be a separate change.